internal/buildsign/pod/common.go

```diff
@@ -35,6 +35,22 @@ var tmpl = template.Must(
 func (pm *buildSignPodManager) podSpec(mld *api.ModuleLoaderData, containerImage string, pushImage bool) v1.PodSpec {
```

Comment:
we have "pm" here, and "mp" in buildsignpodmanager, which is very confusing. Lets reach the common name, it would be better to call it bspm

Reply:
Fixed.